cs/common/vw_common.csproj

```diff
@@ -15,8 +15,8 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
-    <OutputPath>$(SolutionDir)out\target\$(Configuration)\$(PlatformShortName)\</OutputPath>
-    <IntermediateOutputPath>$(SolutionDir)out\int\$(Configuration)\$(PlatformShortName)\$(ProjectName)\</IntermediateOutputPath>
+    <OutputPath>$(SolutionDir)out\target\$(Configuration)\x64\</OutputPath>
```

Comment:
Not really, it just would introduce some oddness about building Win32 going to the x64 directory. I am not sure the 32-bit build works. Would prefer to deal with it in a separate issue, especially since we are doing the work to get Cmake builds to work.